src/main/java/com/dougnoel/sentinel/elements/PageElement.java

```diff
@@ -199,6 +199,16 @@ public PageElement javaScriptSendKeys(String text) {
 
 		return this;
 	}
+	
+	/**
+	 * Moves the mouse to the middle of the element.	
+	 * @return PageElement (for chaining)
+	 */
+	
+	public PageElement mouseOver() {
```

Comment:
Change this to a method that gives us something actionable. Call it getTooltip. Then do the mouseover and return the tooltip instead of the PageElement object. We could do these as separate steps, but it currently gets us nothing, so make this a terminal method.

Reply:
Updated the existing method mouseOver() with the target element and added a new method getToolTipText() to return the text